### PR TITLE
Chart component

### PR DESCRIPTION
**Pull request by josemtzb**
The chart component was created to display evaluation data

### CODE REVIEW

**Review by pmcvtm — CHANGES_REQUESTED** [review]
Overall these look great; we have a couple tweaks and some code cleanup to take care of, and then we can merge this.

File: src/Web/src/components/EvaluationChartComponent/EvaluationChart.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    return (
+        <div className="profile-collapsible-container">
+            <h2 className="profile-collapsible-header" onClick={toggle}>
+                <span className="profile-collapsible-icon">{icon}</span>
+                <span>{title}</span>
+                <span className="profile-collapsible-down-icon"><DownPointingIcon /></span>
+            </h2>
+            <Collapse isOpen={isOpen}>
+            <div class="chartContainer">
+                <span>Year/s </span>
```
Comment (pmcvtm) [inline src/Web/src/components/EvaluationChartComponent/EvaluationChart.js:152]: SHOULD: Change to `Year`

File: src/Web/src/components/EvaluationChartComponent/EvaluationChart.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+            'rgb(31, 142, 243)',
+            'rgb(117, 164, 210)'
+    ]);
+
+    const toggle = () => setIsOpen(!isOpen);
+
+    const [yearsToSelect, setYearsToSelect] = useState([]);
+
+
+    function setTable(title) {
```
[the remark this answers is not in the document]
Comment (josemtzb) [reply src/Web/src/components/EvaluationChartComponent/EvaluationChart.js]: Removed

File: src/Web/src/components/ProfileComponents/Profile.js [hunk 3]
```diff
@@ -4,12 +4,13 @@ import { Nav, NavItem, NavLink} from 'reactstrap';
 import ProfileInfo from './ProfileInfo';
 import CollapsibleTable from './CollapsibleTable';
 import CollapsibleLeaderOfSelf from './CollapsibleLeaderOfSelf';
-import LeaderOfOrgChart from './LeaderOfOrgChart';
+//import LeaderOfOrgChart from './LeaderOfOrgChart';
```
[the remark this answers is not in the document]
Comment (josemtzb) [reply src/Web/src/components/ProfileComponents/Profile.js]: Removed

File: src/Web/src/components/ProfileComponents/Profile.js [hunk 4]
```diff
@@ -30,7 +31,7 @@ const Profile = () => {
                     <NavLink onClick={() => setActiveComponent("general")}>General Info</NavLink>
                 </NavItem>
                 <NavItem className={activeComponent === "leader" ? "current-profile-page nav-option" : "nav-option"}>
-                    <NavLink onClick={() => setActiveComponent("leader")}>Performance</NavLink>
+                    <NavLink onClick={() => setActiveComponent("leader")}>Leader or self</NavLink>
```
[the remark this answers is not in the document]
Comment (josemtzb) [reply src/Web/src/components/ProfileComponents/Profile.js]: Change reverted

File: src/Web/src/components/EvaluationChartComponent/EvaluationChart.js [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+        let dataSets = [];
+        let barNumber = 0;
+        let ratingsByYear = ratings[selectedYear];
+        if (ratingsByYear !== undefined) {
+            ratingsByYear.forEach(element => {
+                let dataSet ={
+                            label: element.category,
+                            backgroundColor: barColors[barNumber],
+                            borderColor: barBorderColors[barNumber],
+                            maxBarThickness:36,
```
[the remark this answers is not in the document]
Comment (josemtzb) [reply src/Web/src/components/EvaluationChartComponent/EvaluationChart.js]: Fixed

File: src/Web/src/components/EvaluationChartComponent/EvaluationChart.js (hunk 1 above)
Comment (josemtzb) [reply src/Web/src/components/EvaluationChartComponent/EvaluationChart.js]: Updated

File: src/Web/src/components/ProfileComponents/Profile.js [hunk 6]
```diff
@@ -40,12 +41,10 @@ const Profile = () => {
...
                     <ProfessionalDevelopmentTable title='Professional Development and Learning Experiences' data={data.professionalDevelopment}/>
                 </div>
-                ) : activeComponent === "leader" && losMappingResult !== null && losMappingResult !== 0 ? (
+                ) : activeComponent === "leader" ? (
                     <div>
-                        { (losMappingResult).map((obj, i) => {
-                            return (<CollapsibleLeaderOfSelf title={obj.categoryTitle} data={obj}/>)
-                            })
-                        }
+                        <EvaluationChart title='Overall' data={data.evaluations} />
```
[the remark this answers is not in the document]
Comment (josemtzb) [reply src/Web/src/components/ProfileComponents/Profile.js]: Charts are loaded dynamically, based on the API data provided

File: src/Web/src/components/ProfileComponents/CanvasChart.js [hunk 7]
```diff
@@ -2,84 +2,85 @@ import React, { useState, useEffect, useRef } from "react";
...
-      borderWidth: 1
-    },
-    {
-      data: [],
-      backgroundColor: [],
-      borderColor: [],
-      borderWidth: 1
-    }
-  ];
+  // var periods = [];
```
[the remark this answers is not in the document]
Comment (josemtzb) [reply src/Web/src/components/ProfileComponents/CanvasChart.js]: Code restored

**Review by pmcvtm — APPROVED** [review]

**Pull request merged**